.gitignore

```diff
@@ -0,0 +1,14 @@
+*.o
```

Comment:
Metamath is not only distributed via GitHub but also via an archive tar file.  In the latter case, a gitignore may overwrite an existing one, because the tar-file contents could be extracted within an existing git environment.

Reply:
In this case, we should adjust the archival process to not include the `.gitignore` file (or the `.git` folder, for that matter). Putting the sources in a `src` folder may help with this.

Reply:
I propose merging this with the .gitignore as is. Currently archive generation is on hiatus since Norm isn't around to keep it running anymore, and when it returns it will be powered by github actions or such and so we will be able to adjust it as necessary.

Reply:
This means we abandon the distribution via a tar file for now.  IMO GitHub is good enough to be the primary (and possibly only) distribution path.  This is still a strategic decision that should be evaluated as an issue first.

Reply:
Ok, I'm done for the day but you should open an issue about this. I agree that GitHub can become the new distribution path. The tar file method is already dead I think, since it's manually triggered and 0.198 is the last version that will be produced that way. You can still produce 0.198 from github sources, but 0.200 will use an as-yet-unwritten tar building script running in CI.

Reply:
I suggest to merge this - there was no real objection #19 after more than a week.